src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/UniversalRemoteAuthenticationProvider.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.ProjectSystem.Debug;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.PropertyPages
+{
+    /// <summary>
+    /// Implementation of IRemoteAuthenticationProvider for the "Universal" authentication type.
+    /// </summary>
+    [Export(typeof(IRemoteAuthenticationProvider))]
+    [AppliesTo("WinUI")] // TODO: ???
+    internal class UniversalRemoteAuthenticationProvider : IRemoteAuthenticationProvider
+    {
+        [ImportingConstructor]
+        public UniversalRemoteAuthenticationProvider(UnconfiguredProject _) // force MEF scope
```

Comment:
These don't need to be scoped to a project - they contain no state, see IItemTypeGuidProvider implementors for an example.

Reply:
The IItemTypeGuidProvider classes have empty ImportingConstructor constructos, are they important? Or also unnecessary?

And also, yes these contain no state, but they do have capabilities so need to be picked up by different projects. Am I right in assuming based on your statement that it doesn't matter what scope the class is in, the capabilities will still be filter appropriately on the Import side? I've always read the attributes in my head as "Export this if these capabilities are present" but I'm guessing thats not the right mental model.

Reply:
They are filtered on the import side.